test/import/FSHImporter.ValueSet.test.ts

```diff
@@ -1244,7 +1209,37 @@ describe('FSHImporter', () => {
         const result = importSingleText(input, 'Insert.fsh');
         const vs = result.valueSets.get('MyVS');
         expect(vs.rules).toHaveLength(1);
-        assertInsertRule(vs.rules[0] as Rule, '', 'MyRuleSet');
+        assertInsertRule(vs.rules[0], '', 'MyRuleSet');
+      });
+
+      it('should parse an insert rule at a concept', () => {
+        const input = leftAlign(`
+        ValueSet: ZooVS
+        * ZOO#hippo
+        * ZOO#hippo insert DesignationRules
+        `);
+        const result = importSingleText(input, 'Insert.fsh');
+        const vs = result.valueSets.get('ZooVS');
+        expect(vs.rules).toHaveLength(2);
+        assertValueSetConceptComponent(vs.rules[0], 'ZOO', undefined, [
+          new FshCode('hippo', 'ZOO').withFile('Insert.fsh').withLocation([3, 3, 3, 11])
+        ]);
+        assertInsertRule(vs.rules[1], '', 'DesignationRules', [], ['ZOO#hippo']);
+        expect(loggerSpy.getAllMessages('error')).toHaveLength(0);
+      });
+
+      it('should log an error when an insert rule has more than one concept', () => {
+        const input = leftAlign(`
+        ValueSet: ZooVS
+        * ZOO#hippo ZOO#big-hippo insert DesignationRules
+        `);
+        const result = importSingleText(input, 'Insert.fsh');
+        const vs = result.valueSets.get('ZooVS');
+        expect(vs.rules).toHaveLength(0);
+        expect(loggerSpy.getAllMessages('error')).toHaveLength(1);
+        expect(loggerSpy.getLastMessage('error')).toMatch(
+          /Only one concept may be listed before an insert rule on a ValueSet\..*File: Insert\.fsh.*Line: 3\D*/s
+        );
```

Comment:
I have a question about this case. We log an error if there are multiple concepts listed before the insert rule, but I don't think we log any error if the caret value rule is included after multiple concepts. For example, something like this doesn't log an errors and adds the designation to the hippo concept. Is that expected?

```
ValueSet: ZooVS
* ZOO#hippo "Hippopotamus"
* ZOO#rhino
* ZOO#hippo ZOO#rhino ^designation.value = "hipopótamo"
```

Reply:
Oh dang. Yeah, multiple concepts in a VS should trigger an error no matter what. Good find.